.github/Dockerfile-cloudrun

```diff
@@ -0,0 +1,18 @@
+FROM pomerium/vals-entrypoint as entrypoint
+
+FROM busybox:latest as build
+RUN touch /config.yaml
+
+FROM gcr.io/distroless/base
```

Comment:
I went back and forth on this.  If we switched to an image with a shell environment, we could make the interface for the config file slightly cleaner, eg "CONFIG=[ref]".  I _think_ sticking with our standard image is worth slightly more complexity but open to other opinions on usability.